### DISCUSSION TITLE
건물 내부 질문

### DISCUSSION THREAD

**Discussion by fnxmxl** [naverz/zepeto-studio-kor#760] [5. Q&A]
안녕하세요 최근에 제페토 월드 제작을 하게 된 뉴비입니다. 현재 내부 전시실을 제작 중인데 이상하게 모바일로 들어가서 보면 한쪽 벽이 색이 다르게 나오며 접속시 벽의 색이 달라지는 위치가 랜덤으로 변합니다. 처음에는 조명을 사용해서 문제가 생겼나 싶었는데 아직까지 이유를 정확히 모르겠습니다. 혹시 아래 이미지에서 나온 것에서 주변 벽들과 색을 같게 할 수 있는 방법이 있을까요? 
![제페토 월드](https://user-images.githubusercontent.com/105581938/168463568-cd8bf318-3a15-4833-9b1b-b9a82ee09554.jpg)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 위 이슈에 대해 도움을 드리고 싶은데, 스크린샷 만으로는 파악하기가 어렵습니다.
프로젝트 파일을 공유주시거나 관련 스크린샷 및 코드 첨부 부탁 드립니다.
감사합니다.

**Reply by mayalhc** [reply]
한쪽 면이 뒤집어 진거 같습니다...
